ethpm_types/contract_type.py

```diff
@@ -150,38 +171,50 @@ class SourceMap(BaseModel):
 
     __root__: str
 
+    def __repr__(self) -> str:
+        return self.__root__
+
+    def __str__(self) -> str:
+        return self.__root__
```

Comment:
Isn't this the default `__str__` method output if `__repr__` is defined?

Reply:
It didn't seem like it. I added the test before I added this and it would fail at first and give me this:

```
SourceMap(__roor__"<realsourcemap>")
```

Reply:
where `realsourcemap` was the real source map

Reply:
If I comment out `def __str__`, I get this:

```
assert "__root__='44...441:16;;;;;;'" == '449:4441:16:...4441:16;;;;;;'
```

so it is `__root__='44...441:16;;;;;;'`.

Does overriding `__str__` here cause any issues I am not yet aware of?